lib/pbench/server/__init__.py

```diff
@@ -143,6 +117,101 @@ def mocked_time():
             + " ".join([f"WONT-INDEX.{i:d}" for i in range(1, 12)])
         )
 
+    @property
+    def server_config(self):
+        return self.conf["pbench-server"]
+
+    @property
+    def logger(self):
```

Comment:
This seems to be adding a new `logger` attribute (property) to a `PbenchServerConfig` class.  All the existing callers create their own loggers from the server config.  Can we have the API instance do the same?

Reply:
Addressed.